virttest/qemu_devices/qcontainer.py

```diff
@@ -2246,6 +2246,8 @@ def chardev_define_by_params(self, chardev_id, params, file_name=None):
         if backend in ['unix_socket', 'file', 'pipe', 'serial',
                        'tty', 'parallel', 'parport']:
             chardev_param.update({'path': file_name})
+            if (backend == 'pipe' and params.get('auto_create_pipe', 'yes') == "yes"):
```

Comment:
if  auto_create_pipe , what will happened?   I suggest to change code to :

import os, stat

if backend == 'pipe':
    auto_pipe = params.get('auto_create_pipe')
    if auto_pipe == "yes":
        process.system('mkfifo %s' % filename)
    elif not stat.S_ISFIFO(os.stat(filename).st_mode):
        raise  some exception`
`

Reply:
@xutian thanks, we involve `auto_create_pipe`, since in some case vm need boot up without the pipe file `filename`, this is why it can not raise exception when `not stat.S_ISFIFO(os.stat(filename).st_mode)`
such as:
some case test, create pipe `filename.in`, `filename.out`, and boot vm with `path=filename`. If there have pipe `filename`, it will failed.

Reply:
OK